src/views/certificate.ejs

```diff
@@ -39,7 +39,13 @@
             style = "height: 100% ;width: 100% ;padding: 0px;margin: 0px;	" > 
         </img> 
         <div style = "position: absolute;top: 0;left: 0;height: 1155px;width: 1500px;padding: 0px;margin: 0px;">
-            <div style="position: absolute; right: 50px; margin: 10px; color: rgb(224 68 44); font-family: monospace;">Certified On: <%= data.createdAt.toISOString().split('T').join(' ').slice(0,19) %></div>
+            <div

```

Comment:
- Put this in bottom left.
- Make ID readable like `67vj7-sdkvb-skvjs-sdkjv` add - after every 5 characters.